### DISCUSSION TITLE
Motion home screen not showing video, only camera icons.

### DISCUSSION THREAD

**Discussion by fractal815** [Motion-Project/motion#1827] [Q&A] [answered]
Not really sure what happened that cameras are not displaying.   System has been running for years, and had worked previously.   Perhaps an OS (Kubuntu 24.04) update has caused a problem.   Video recording is working properly.   Not seeing any unusual errors in log.    
Kubuntu 24.04   Motion 4.6.0.

**Comment by Mr-Dave** [comment, marked as answer]
Make sure you do not accidentally have two instances of Motion running.

**Reply by fractal815** [reply]
Thanks Mr-Dave....   Don't know how that happened but that was it!
